plugins/rocks.xml.webhelp/js/xml.rocks.js

```diff
@@ -47,43 +47,30 @@ function expandCollapseAll() {
 }
 
 //Create PDF from HTML topic content
-function getPDF() {
-    var idTopicArticle = document.getElementById("topic-article");
-    var removedChild = idTopicArticle.querySelector('.related-links');
+var idTopicArticle = document.getElementById("topic-article");
+var removedChild = idTopicArticle.querySelector('.related-links');
 
-    idTopicArticle.removeChild(idTopicArticle.querySelector('.related-links'));
+idTopicArticle.removeChild(idTopicArticle.querySelector('.related-links'));
 
-    var htmlWidth = $("#topic-article").width();
-    var htmlHeight = $("#topic-article").height();
+var topicName = document.getElementsByClassName("title topictitle1")[0].textContent;
 
-    var topLeftMargin = 15;
-    var pdfWidth = htmlWidth + (topLeftMargin * 2);
-    var pdfHeight = (pdfWidth * 1.5) + (topLeftMargin * 2);
-    var canvasImageWidth = htmlWidth;
-    var canvasImageHeight = htmlHeight;
+topicName = topicName.replace(/\s+/g, '-');
 
-    var totalPDFPages = Math.ceil(htmlHeight / pdfHeight) - 1;
 
-    var topicName = document.getElementsByClassName("title topictitle1")[0].textContent;
-
-    topicName = topicName.replace(/\s+/g, '-');
-
-    html2canvas(idTopicArticle, {allowTaint: true}).then(function (canvas) {
-        canvas.getContext('2d');
-        var imgData = canvas.toDataURL("image/jpeg", 1.0);
-        var pdf = new jsPDF('p', 'pt', [pdfWidth, pdfHeight]);
-        pdf.addImage(imgData, 'JPG', topLeftMargin, topLeftMargin, canvasImageWidth, canvasImageHeight);
-
-        for (var i = 1; i <= totalPDFPages; i++) {
-            pdf.addPage(pdfWidth, pdfHeight);
-            pdf.addImage(imgData, 'JPG', topLeftMargin, -(pdfHeight * i) + (topLeftMargin * 4), canvasImageWidth, canvasImageHeight);
-        }
-
-        pdf.save(topicName + ".pdf");
-    });
-    // Append 'related-links' to the page after downloading current topic
+function ExportPdf() {
+    kendo.drawing
+        .drawDOM("#topic-article",
+            {
+                paperSize: "A4",
+                margin: {top: "1cm", bottom: "1cm", right: "1cm", left: "1cm"},
+                scale: 0.8,
+                height: 500
+            })
+        .then(function (group) {
+            kendo.drawing.pdf.saveAs(group, topicName + ".pdf");
```

Comment:
Looks like topicName is incorrect. I get 'undefined.pdf' filename

Reply:
Also, we get console log errors:
![image](https://user-images.githubusercontent.com/4980278/149798465-7ef67d37-719c-4549-a688-b9393df40be8.png)